signprot/templates/signprot/coupling_browser2.html

```diff
@@ -0,0 +1,401 @@
+                    </ul>
+                </div>
+
+            <div style="width:100%; display:inline;">
+                <div class="btn-group" style="padding-left:10px;">
+                    <a id="dlink" style="display:none;"></a>
+                    <div id="excel_table" style2="display:none;"></div>
+                    <input class="btn btn-primary btn-sm" type="button" id="toggle_fixed_btn2" value="Disable fixed columns"
+                           href="javascript:void(0)" data-toggle="buttons">
+                    <input class="btn btn-primary btn-sm" type="button" onclick="resetHidden2()" id="toggle"
+                           value="Show hidden columns">
+                </div>
+
+                <div class="btn-group" style="padding-left:10px;">
+                    <input class="btn btn-primary btn-sm" type="button"
+                           onclick="tableToExcel('subtypestabletab', 'Subtypes data', 'subtypes_coupling.xls')"
+                           value="Export Excel">
+                    <input class="btn btn-primary btn-sm" type="button" onclick="reset_tab2()" value="Reset All">
+                </div>
+            </div>
+
+            <table width="100%" class="display compact text-nowrap" id="subtypestabletab" >
+                <thead>
+                <tr>
+                    <th colspan=1 style='height:35px; text-align:left;'>
+                        Source
+                    </th>
+                    <th colspan=4 style='height:35px; text-align:left;'>
+                        Receptor
+                    </th>
+                    <th colspan=4 style='height:35px; text-align:left;'>
+                        Guide to Pharmacology
+                        <button type="button" class="close hide_columns2" columns="5,6,7,8"
+                                style="float:right;display:inline;" aria-label="Close"><span aria-hidden="true">&times;</span>
+                        </button>
+                    </th>
+                    <th colspan=13 style='height:35px; text-align:left;'>
+                        log(Emax/EC50)
+                        <button type="button" class="close hide_columns2" columns="9,10,11,12,13,14,15,16,17,18,19,20,21"
+                                style="float:right;display:inline;" aria-label="Close"><span aria-hidden="true">&times;</span>
+                        </button>
+                    </th>
+                    <th colspan=13 style='height:35px; text-align:left;'>
+                        pEC50
+                    </th>
+                    <th colspan=14 style='height:35px; text-align:left;'>
+                        Emax
+                    </th>
+                </tr>
+
+                <tr>
+                    <th></th>
+                    <th>Cl</th>
+                    <th>ResFam</th>
+                    <th>Uniprot</th>
+                    <th>IUPHAR</th>
+
+                    {% for p, subval in header.items  %}
```

Comment:
If you want to make these subheadings more concise you can use this trick to do a counted for loop:
https://stackoverflow.com/questions/1107737/numeric-for-loop-in-django-templates